easybuild/tools/toolchain/toolchain.py

```diff
@@ -120,7 +120,6 @@ def __init__(self, name=None, version=None, mns=None, class_constants=None, tcde
         """
         self.base_init()
 
-        self.dependencies = []
```

Comment:
@bartoldeman We can't just drop this, since existing easyblocks are using this, see `generic/craytoolchain.py` and `wien2k.py`...